### PR TITLE
HBASE-24739 [Build] branch-1's build seems broken because of pylint

### PR DESCRIPTION
**Pull request by Reidddddd**
Try to use the latest pylint version that supports python2.7.

### CODE REVIEW

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2065/1/console in case of problems.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m  0s |  Docker mode activated.  |
| -1 :x: |  docker  |   5m 25s |  Docker failed to build yetus/hbase:7cf97016b9.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hbase/pull/2065 |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2065/1/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2065/2/console in case of problems.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m  0s |  Docker mode activated.  |
| -1 :x: |  docker  |   0m 11s |  Docker failed to build yetus/hbase:7cf97016b9.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hbase/pull/2065 |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2065/2/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Reidddddd** [comment]
seems not right, hmm...

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2065/3/console in case of problems.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |  12m 29s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  hadolint  |   0m  0s |  hadolint was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   2m 23s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  8s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  shellcheck  |   0m  1s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  0s |  ASF License check generated no output?  |
|  |   |  15m 53s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.12 Server=19.03.12 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2065/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2065 |
| Optional Tests | dupname asflicense hadolint shellcheck shelldocs |
| uname | Linux 9f1fd0cb07e2 4.15.0-101-generic #102-Ubuntu SMP Mon May 11 10:07:26 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-2065/out/precommit/personality/provided.sh |
| git revision | branch-1 / 0841f12 |
| Max. process+thread count | 36 (vs. ulimit of 10000) |
| modules | C:  U:  |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2065/3/console |
| versions | git=1.9.1 maven=3.0.5 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Reidddddd** [comment]
ping @busbey 
I'm not sure if I did it right, would you mind taking a look?

**Comment by busbey** [comment]
seems reasonable. delete the whole "install pylint via pip" section? maybe put up a test commit on this PR that touches a python file so we can ensure pylint works?

File: dev-support/docker/Dockerfile [hunk 1]
```diff
@@ -114,7 +115,7 @@ ENV FINDBUGS_HOME /opt/findbugs
 ####
 # Install pylint
 ####
-RUN pip install pylint==1.9.2
+# RUN pip install pylint==1.9.5
```
Comment (virajjasani) [inline dev-support/docker/Dockerfile:118]: nit: remove entire comment

**Comment by lhofhansl** [comment]
Let's get it in? It's holding up other changes for branch-1

**Comment by Apache-HBase** [comment]
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2065/4/console in case of problems.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   6m 52s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  hadolint  |   0m  0s |  hadolint was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   2m 26s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m  9s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +0 :ok: |  asflicense  |   0m  0s |  ASF License check generated no output?  |
|  |   |  10m 35s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.12 Server=19.03.12 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2065/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2065 |
| Optional Tests | dupname asflicense hadolint shellcheck shelldocs |
| uname | Linux 7f9abd6e7c0c 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-2065/out/precommit/personality/provided.sh |
| git revision | branch-1 / f5080eb |
| Max. process+thread count | 41 (vs. ulimit of 10000) |
| modules | C:  U:  |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-2065/4/console |
| versions | git=1.9.1 maven=3.0.5 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**